README.md

```diff
@@ -1 +1 @@
-testing semantic versioning 4.21.0
```

Comment:
The code changes in this diff show an update from version 4.21.0 to version 5.0.0 for testing semantic versioning. This indicates a major version change, which typically signifies significant changes or new features in the software. It is important to review the release notes for version 5.0.0 to understand the specific changes and updates that have been made.